scripts/postinstall.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Generate preinstalled.json from folders in src/extensions/preinstalled-extensions
```

Comment:
Why it's so complicated?
Why generate `preinstalled.json` from a manual `.zip`? We should dynamically create the `.zip` from `preinstalled.json`.

You can use https://github.com/akhileshthite/chrome-extension-fetch library in `postinstall.js`. 

So, `preinstalled-extensions` directory should only have `preinstalled.json` and it will fetch `.zip` by using `chrome-extension-fetch` with `postinstall.js` script.